src/ui/index.js

```diff
@@ -124,13 +128,72 @@ export const Button = styled.button`
   ${height}
   ${border}
   ${zIndex}
+  ${opacity}
   ${fontSize}
   ${boxShadow}
   ${borderColor}
   ${borderRadius}
+  cursor: ${({ cursor }) => or(cursor, 'initial')};
+  &::before {
```

Comment:
move it to custom button component that inherited(`styled(Button)`) from this button